c/calcite-avatica/calcite-avatica_ubi_8.sh

```diff
@@ -0,0 +1,29 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : calcite-avatica
+# Version       : master(945d06c2a)
+# Source repo   : https://github.com/apache/calcite-avatica.git
+# Tested on     : RHEL 8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer    : sachin.kakatkar@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+PACKAGE_NAME=calcite-avatica
+PACKAGE_VERSION=945d06c2a
```

Comment:
please try the build for latest stable / requested version and accept it as an input argument

Reply:
The latest code is on branch master. other release versions are very old and they do not use gradle. i have targeted specific commit from the master.

Reply:
https://github.com/apache/calcite-avatica/tree/rel/avatica-1.18.0 seems to me like it uses gradle?